aspnetcore/blazor/security/server/index.md

```diff
@@ -308,28 +308,12 @@ public class ExampleService
 }
 ```
   
-Register the service as scoped. In a server-side Blazor app, scoped services have a lifetime equal to the duration of the client connection [circuit](xref:blazor/hosting-models#blazor-server).
-
-:::moniker range=">= aspnetcore-6.0"
-
-In the `Program` file:
+In the `Program` file, the `ExampleService` is registered as a scoped service. In a server-side Blazor app, scoped services have a lifetime equal to the duration of the client connection [circuit](xref:blazor/hosting-models#blazor-server).
```

Comment:
The sample uses `ExampleService` only with `OwningComponentBase`, and stating that "scoped services have a lifetime equal to the duration of the client connection circuit" is confusing in this context. While this might sometimes be true (when using regular dependency injection where the scope aligns with the circuit), in our case, the lifetime is influenced by the `OwningComponentBase` scoping.

I’m not sure how to resolve this, but for the average reader, this will be a headache.

Reply:
The phrase "In a server-side Blazor app" is meant to explain that this is ***normally*** the case. I'll adjust the text to make that clear.